localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -301,6 +302,8 @@ def return_response(self, method, path, data, headers, response):
         elif action == 'PutItem':
             if response.status_code == 200:
                 keys = dynamodb_extract_keys(item=data['Item'], table_name=table_name)
+                # Get stream specifications details for the table
+                stream_spec = dynamodb_get_table_stream_specification(table_name=table_name)
```

Comment:
nit: Let's move this line below the `if` check to line 309 (minor performance improvement, avoiding unnecessary calls).